common/pango.c

```diff
@@ -104,6 +106,7 @@ void get_text_size(cairo_t *cairo, const PangoFontDescription *desc, int *width,
 void get_text_metrics(const PangoFontDescription *description, int *height, int *baseline) {
 	cairo_t *cairo = cairo_create(NULL);
 	PangoContext *pango = pango_cairo_create_context(cairo);
+	pango_context_set_round_glyph_positions(pango, false);
```

Comment:
We also need to add this to the rendering path on line ~130

Reply:
Indeed! I didn't realize there was an entire abstraction between sway_text_node and pango!